manifests/repo.pp

```diff
@@ -1,49 +1,53 @@
-# private class
-class sensu::repo {
+# @summary Private class to manage sensu repository resources
+# @api private
+#
+class sensu::repo (
+  $manage_repo = $sensu::manage_repo
```

Comment:
I'd recommend moving the variable outside parameter list.

Reply:
I did it, and now it's more close to the coding style of the rest of the module

Reply:
done